test/remote/account_preverified_token_tests.js

```diff
@@ -4,11 +4,11 @@
 
 var test = require('../ptaptest')
 var TestServer = require('../test_server')
-var Client = require('../client')
+var Client = require('../client/index')
 var JWTool = require('fxa-jwtool')
 
 process.env.TRUSTED_JKUS = 'http://127.0.0.1:9000/.well-known/public-keys'
-var config = require('../../config').getProperties()
+var config = require('../../config/index').getProperties()
```

Comment:
Not 100% sure, but think it was added in because the tests were failing originally with sign-in confirmation. Retested and it works without it, removing.